### ISSUE TITLE
Delpoy only binary files without source code zip

### ISSUE THREAD

**Issue by RareScrap** [travis-ci/travis-ci#9959]
My project was written in Java and uses Gradle. I have two problems.
Firstly, Jar-files do not publish on Github Realises. Does not work at all. Here travis config
`
language: java
sudo: false
install: gradle wrapper --gradle-version 4.2
jdk:
- oraclejdk8
script:
- chmod +x gradlew
- "./gradlew test"
- "./gradlew build"
- "./gradlew jar"
- cd build
- ls
- cd libs
- ls
- pwd
cache:
  directories:
  - "$HOME/.gradle"
  - ".gradle"
deploy:
  provider: releases
  api_key:
    secure: [MY_KEY]
  file: "$HOME/travis/build/libs/travis.test-1.0-SNAPSHOT.jar"
  skip_cleanup: true
  file_glob: true
  on:
    repo: RareScrap/travis.test
    branches:    # ← new!
          only:      # ← new!
            - master # ← new!
`
Instead, only archives with the source code are published. And this is the second problem. How do I publish ONLY the binaries?
![2018-08-08_00-28-22](https://user-images.githubusercontent.com/7997602/43789263-10c65d9a-9aa2-11e8-96b9-cdf72457df4e.png)

**Comment by RareScrap** [comment]
bump

**Comment by onkarshinde** [comment]
@RareScrap Are you sure the path you have specified in file key is correct? Isn't file supposed to be a path relative to your workspace?

**Comment by RareScrap** [comment]
@onkarshinde You're right. The path is not correct. I corrected it on the ```file: "/home/travis/build/RareScrap/SavageWorldRP/build/libs/*"``` (looks ugly, does not it?) And now the binaries are successfully published. However, the damned archives are published too!

**Comment by onkarshinde** [comment]
Not sure why you are using the absolute path. Try using path relative to your workspace.

As for the archives problem I don't think travis has any control over it. That seems to be GitHub feature. If you manually create a release against any tag that also will automatically add source archives.
